resources-generator/src/main/kotlin/dev/icerock/gradle/generator/PluralsGenerator.kt

```diff
@@ -77,7 +78,36 @@ abstract class PluralsGenerator(
             mutableMap[name] = pluralMap
         }
 
-        return mutableMap
+        return processLanguagePlurals(mutableMap)
+    }
+
+    /**
+     * Called after the language plurals have been loaded.
+     */
+    protected open fun processLanguagePlurals(languagePlurals: Map<KeyType, PluralMap>): Map<KeyType, PluralMap> {
```

Comment:
but now we have only one common keys type - `zero`, `one`, `two`, `many`, `few`, `other`.
i think this code not required anymore.